code/game/gamemodes/traitor/double_agents.dm

```diff
@@ -1,14 +1,11 @@
 /datum/game_mode/traitor/double_agents
 	name = "double agents"
 	config_tag = "double_agents"
-	restricted_jobs = list("Cyborg", "AI", "Captain", "Head of Personnel", "Chief Medical Officer", "Research Director", "Chief Engineer", "Head of Security") // Human / Minor roles only.

```

Comment:
Err, you sure that you will not cause some sort of wonkiness such as DA Captain/HoS/Borgs, or are their inability to be antags handled someplace else?